addons/l10n_latam_check/tests/test_own_checks.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo.addons.l10n_latam_check.tests.common import L10nLatamCheckTest
+from odoo.tests.common import tagged
+
+
+@tagged('post_install_l10n', 'post_install', '-at_install')
+class TestOwnChecks(L10nLatamCheckTest):
+
+    def test_01_pay_with_multiple_checks(self):
+        """ Create one chec first check should choose deferred check by default. On current check force a different
+        number than next one """
```

Comment:
Double check the meaning/spelling of the sentence, I'm not sure what you mean.
Also, it seems some asserts are missing
> force a different number than next one

Where is this checked?

> should choose deferred check by default

Where is this checked?

Reply:
Hi @william-andre 

We have updated the unit test in order to have sense.
* remove the part "On current check force a different number than next one" from the docstring because this was related to checkbooks and actually is not part of the functionality.
* update the unit tests code itself in order to be able to check that an own check payment can be created and posted without problem,
* we use Form() to simulate the check creation from view and we check that we actually are a deferred check type
 
you can check the changes in 46374220709afb90f113d0083b7513d3eabf3d9a 

cc @jjscarafia @jco-odoo